styles.css

```diff
@@ -0,0 +1,224 @@
+*{
+	box-sizing: border-box;
+}
+
+html{
+	font-size: 20px;
+}
+
+.container {
+	width: 1600px;
+	margin: 0 auto;
+}
+
+
+/* navbar section*/
+nav{
+	display: flex;
+	background: #E5E2DE;
+	height: 60px;
+	margin: 10px 0;
+	justify-content: center;
+	align-items: center;
+	
+}
+
+nav > div{
+	display: flex;
+}
+
+.navigation{
+	padding: 5px;
+	margin: 5px;
+		
+}
+
+.navBorder{
+	border: 2px solid #777674;
+}
+
+.navigation > a{
+	text-decoration: none;
+}
+
+.navigation > a:visited{
```

Comment:
remove - just add color: #000000 to the navigation > a selector above

Reply:
I checked again and you're right.  It just doesn't like having a single 0 after the hash.